content/influxdb/cloud-serverless/get-started/setup.md

```diff
@@ -158,26 +158,39 @@ or by environment variables.
 {{% tab-content %}}
 <!----------------------------- BEGIN API CONTENT ----------------------------->
 
-When using the InfluxDB API, provide the required connection credentials in the
-following ways:
+cURL and client library examples in this getting started tutorial assume you have
+environment variables assigned to your InfluxDB credentials.
 
-- **InfluxDB host**: [InfluxDB Cloud Serverless region URL](/influxdb/cloud-serverless/reference/regions/)
-- **InfluxDB API Token**: Include an `Authorization` header that uses either 
-  `Bearer` or `Token` scheme and your InfluxDB API token. For example:  
-  `Authorization: Bearer 0xxx0o0XxXxx00Xxxx000xXXxoo0==`.
-- **InfluxDB organization name or ID**: Depending on the API endpoint used, pass
-  this as part of the URL path, query string, or in the request body.
-
-All API examples in this tutorial use **cURL** from a command line.
-To provide all the necessary credentials to the example cURL commands, set
-the following environment variables in your command line session.
+To assign environment variables to your credentials, enter the
+following commands into your profile settings or terminal:
 
 ```sh
-export INFLUX_HOST=https://cloud2.influxdata.com
-export INFLUX_ORG=<YOUR_INFLUXDB_ORG_NAME>
-export INFLUX_ORG_ID=<YOUR_INFLUXDB_ORG_ID>
-export INFLUX_TOKEN=<YOUR_INFLUXDB_API_TOKEN>
+export INFLUX_URL=https://cloud2.influxdata.com
+export INFLUX_HOST=cloud2.influxdata.com
+export INFLUX_ORG=ORG_NAME
+export INFLUX_ORG_ID=ORG_ID
+export INFLUX_TOKEN=API_TOKEN
 ```
+
+Replace the following:
+
+- **`ORG_NAME`**: your InfluxDB organization name
+- **`ORG_ID`**: your InfluxDB organization ID
+- **`API_TOKEN`**: your InfluxDB API token with sufficient permissions to write or query
+  data in your bucket
+
+Keep the following in mind when using API clients and client libraries:
+
+- InfluxDB ignores `org` and `org_id` parameters in API write and query requests,
+  but some clients still require them.
+- In code samples, **host** usually refers to the hostname
+  (`cloud2.influxdata.com`, without the `https://`) part of your
+  [InfluxDB Cloud Serverless region URL](/influxdb/cloud-serverless/reference/regions/).
```

Comment:
This may be confusing since I don't believe `cloud2.influxdata.com` will get replaced on page load since it's inline code and not in a code block.

```suggestion
- In code samples, **host** usually refers to your
  [InfluxDB Cloud Serverless region URL](/influxdb/cloud-serverless/reference/regions/)
  (without `https://`).
  .
```